serveradmin/resources/views.py

```diff
@@ -111,13 +110,15 @@ def index(request):
             hosts[server['hostname']] = dict(server)
 
     page = int(request.GET.get('page', 1))
-    per_page = int(request.GET.get('per_page', 8))
+    per_page = int(request.GET.get(
```

Comment:
Same here, what will happen if per_page is negative or 0?

Reply:
I set a min value so that it is not possible. If the user does so by manipulating the URL nothing happens but the paginator show a negative value.